requeueFailedCRRCronJob.js

```diff
@@ -0,0 +1,284 @@
+const { waterfall, doWhilst, eachLimit, retry } = require('async');
+const http = require('http');
+const { Producer } = require('node-rdkafka');
+const { scheduleJob } = require('node-schedule');
+
+const { errors } = require('arsenal');
+const VID_SEP = require('arsenal').versioning.VersioningConstants
+      .VersionId.Separator;
+const { Logger } = require('werelogs');
+
+const BackbeatClient = require('./BackbeatClient');
+
+const log = new Logger('s3utils::requeueFailedCRRCronJob');
+const {
+    CLOUDSERVER_ENDPOINT,
+    BACKBEAT_API_ENDPOINT,
+    ACCESS_KEY,
+    SECRET_KEY,
+    SITE_NAME,
+    KAFKA_HOSTS,
+    KAFKA_TOPIC,
+} = process.env;
+const CRON_RULE = process.env.CRON_RULE || '0 */6 * * *';
+
+
+if (!CLOUDSERVER_ENDPOINT) {
```

Comment:
`ReferenceError` could be used here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError

Reply:
I am usually wary of reusing error types that have a specific meaning in the language for other purposes. The check we do here is kinda close to a variable existence check but it's more of a pre-validation of arguments passed to the script (via environment vars) than a real reference error, hence I prefer to use generic errors with an explicit message.